docs/pre-release/faqs.md

```diff
@@ -99,7 +99,11 @@ Running this command will also stop the local daemon running.
 
 ** How does Telepresence connect and tunnel into the Kubernetes cluster?**
 
- The connection between your laptop and cluster is established via the standard `kubectl` mechanisms and SSH tunnelling.
+The connection between your laptop and cluster is established by using
+the `kubectl port-forward` machinery (though without actually spawning
```

Comment:
I would maybe say something like "Without actually requiring the `kubectl` cli to spawn a separate program" just to make it abundantly clear that `kubectl` is not needed for telepresence.